layouts/shortcodes/ct_logs.html

```diff
@@ -1,8 +1,23 @@
-{{ range $.Site.Data.transparency.transparencyLogs }}
-<p>
-  <strong>{{ i18n "certificate_transparency_name" }}{{ .Site.Params.beforeColon | safeHTML }}: {{ .name }}</strong><br>
-  {{ i18n "certificate_transparency_role" }}{{ .Site.Params.beforeColon | safeHTML }}: {{ if eq .role "production" }}{{ i18n "certificate_transparency_role_production" }}{{ end }}{{ if eq .role "test" }}{{ i18n "certificate_transparency_role_test" }}{{ end }}<br>
-  {{ i18n "certificate_transparency_uri" }}{{ .Site.Params.beforeColon | safeHTML }}: {{ .uri }}<br>
-  {{ i18n "certificate_transparency_public_key" }}{{ .Site.Params.beforeColon | safeHTML }}: <code>{{ .publicKey }}</code>
-</p>
+{{ $inner := .Inner }}
+{{ $data := .Get "data" }}
+{{ range $.Site.Data.transparency.categories }}
+    {{ $category := . }}
+    {{ if (eq $category $data "$category") }}
+    {{ $i18nVar := (printf "%s%s" "certificate_transparency_role_" $data) }}
+        <h3>{{ i18n $i18nVar }}</h3>
```

Comment:
I have a personal preference for a simple basic `if`/`else if`, it's easier to read and it allows grep in the code (to find where a sentence is used for example)

Reply:
Would you prefer passing in the i18n variable from `content/en/docs/ct-logs.html` and removing the printf? Would you accept having a comment above `$i18nVar` like `{{/* certificate_transparency_role_production or certificate_transparency_role_testing *\}}`?

Reply:
Instead of

```html
    {{ $i18nVar := (printf "%s%s" "certificate_transparency_role_" $data) }}
        <h3>{{ i18n $i18nVar }}</h3>
```

Simply

```html
    {{ if eq $data "test" }}
        <h3>{{ i18n "certificate_transparency_role_test"}}</h3>
    {{ else }}
        <h3>{{ i18n "certificate_transparency_role_production"}}</h3>
    {{ end }}
```